arch/arm/include/aarch32/cortex_m/tz_ns.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief
```

Comment:
@oyvindronningstad  some info is needed here

Reply:
Added